website/integrations/services/engomo/index.mdx

```diff
@@ -0,0 +1,87 @@
+---
+title: Integrate with engomo
+sidebar_label: engomo
+---
+
+# Integrate with engomo
+
+<span class="badge badge--secondary">Support level: Community</span>
+
+## What is engomo
+
+> engomo is an low-code app development platform to create enterprise apps for smartphones and tablets based on Android, iOS, or iPadOS.
+>
+> -- https://engomo.com/
+>
+> This guide explains how to set up engomo to use authentik as the OAuth provider for the application login on the smartphone/tablet and login to the admin WebGUI (composer).
```

Comment:
```suggestion
> This guide explains how to set up engomo to use authentik as the OAuth provider for the application on the smartphone/tablet and allow logging in to the admin WebGUI (composer).
```

Reply:
Did that rewrite leave the original intent? The sentence was a little confusing.